pallets/delegating/src/lib.rs

```diff
@@ -441,12 +441,13 @@ impl<T: Trait> CreditDelegateInterface<T::AccountId, BalanceOf<T>, PositiveImbal
         let validator_leftover_payout = era_payout - validator_commission_payout;
 
         // We can now make total validator payout:
-        if let Some(imbalance) =
-            Self::make_payout(validator_payee.clone(), validator_commission_payout)
-        {
-            Self::deposit_event(RawEvent::Reward(validator_payee, imbalance.peek()));
+        if validator_commission_payout != <BalanceOf<T>>::default() {
+            if let Some(imbalance) =
+                Self::make_payout(validator_payee.clone(), validator_commission_payout)
+            {
+                Self::deposit_event(RawEvent::Reward(validator_payee, imbalance.peek()));
```

Comment:
is the imbalance the payout to validator?

Reply:
yes